negative.py

```diff
@@ -0,0 +1,22 @@
+# Import the dependencies
+import argparse
+import cv2 #image processing lib
+import numpy as np #matrix manipulation
+from time import sleep #for slowing down the process to make progress visible 
+from tqdm import tqdm as tqdm # for progess bar 
```

Comment:
I guess we don't need the progress bar here as there is not much computationally expensive operations being done

Reply:
Ok,will change..